src/test/javascript/spec/component/exam/participate/exam-participation-cover.component.spec.ts

```diff
@@ -166,6 +181,39 @@ describe('ExamParticipationCoverComponent', () => {
         jest.advanceTimersByTime(UI_RELOAD_TIME + 1); // simulate setInterval time passing
         expect(component.waitingForExamStart).toBe(true);
         expect(component.timeUntilStart).toBe('');
+
+        // Case TestExam
+        component.testRun = false;
+        component.testExam = true;
+        component.exam.testExam = true;
+        const exercise1 = { id: 87, type: ExerciseType.TEXT } as Exercise;
+        component.studentExam.exercises = [exercise1];
+
+        jest.spyOn(examParticipationService, 'loadStudentExamWithExercisesForConductionOfTestExam').mockReturnValue(of(studentExam));
+
+        component.exam.startDate = dayjs().subtract(1, 'days');
+
+        component.startExam();
+        tick();
+        expect(component.studentExam).toEqual(studentExam);
+
+        const startDate1 = dayjs();
+        const now1 = dayjs();
+        component.exam.startDate = startDate1.add(2, 'hours');
+        jest.spyOn(artemisServerDateService, 'now').mockReturnValue(now1);
+        component.startExam();
+        tick();
+        jest.advanceTimersByTime(UI_RELOAD_TIME + 1); // simulate setInterval time passing
+        expect(component.waitingForExamStart).toBe(true);
```

Comment:
Please decide if you want to use ``toBe(true)`` vs. ``toBeTrue()`` and  ``toBe(false)`` vs. ``toBeFalse()``and use it consistently in all your test files (I'd recommend the latter)